packages/icons/src/fossa.svg

```diff
@@ -1,5 +1,191 @@
-    </g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="100%" viewBox="0 0 286 284" enable-background="new 0 0 286 284" xml:space="preserve">
```

Comment:
The `width` needs to be removed along with `x` and `y`. I think you need to run these through a SVG optimiser (I like [ImageOptim](https://imageoptim.com/mac))

Reply:
Looking into the visuals.

Reply:
Received sign-off from PM to go with this logo.